spec/ruby-push-notifications/mpns/mpns_connection_spec.rb

```diff
@@ -0,0 +1,46 @@
+
+module RubyPushNotifications
+  module MPNS
+    describe MPNSConnection do
+
+      describe '::post' do
+
+        let(:body) { 'abc' }
+        let(:headers) {
+          {
+            'x-notificationstatus' => ['Received'],
```

Comment:
Yes.. this is because response.to_hash method
https://github.com/calonso/ruby-push-notifications/pull/3/files#diff-34dd4ad17eae8e4acbac88092adfca0dR51

http://ruby-doc.org/stdlib-2.2.1/libdoc/net/http/rdoc/Net/HTTP.html#class-Net::HTTP-label-Response+Data